QamarDeen/QamarDeen/Model/Generated/Day+CoreDataProperties.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Day+CoreDataProperties.swift
+//  QamarDeen
+//
+//  Created by Mazyad Alabduljaleel on 12/25/16.
+//  Copyright © 2016 Batoul Apps. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+
+extension Day {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<Day> {
+        return NSFetchRequest<Day>(entityName: "Day");
+    }
+
+    @NSManaged public var hijriDate: NSDate?
```

Comment:
Perhaps we should consider storing a current UTC offset, this will help us in re-creating the calendar date so that we avoid confusion if the user travels or when DST occurs.